src/applications/claims-status/components/claim-status-tab/RecentActivity.jsx

```diff
@@ -27,17 +27,10 @@ export default function RecentActivity({ claim }) {
     cstClaimPhasesEnabled &&
     isDisabilityCompensationClaim(claim.attributes.claimTypeCode);
 
-  const is5103Notice = item => {
-    return (
-      isAutomated5103Notice(item.displayName) ||
-      item.displayName === 'Review evidence list'
-    );
-  };
-
   const getTrackedItemDescription = item => {
     const displayName =
-      is5103Notice(item) && cst5103UpdateEnabled
-        ? '5103 Evidence Notice'
+      cst5103UpdateEnabled && is5103Notice(item.displayName)
+        ? 'List of evidence we may need (5103 notice)'
```

Comment:
As noted in the PR description, we were previously only overriding the display name for automated 5103 notices from the API and for mocked standard 5103 notices. Now we also override the display name for closed standard 5103 notices from the API. The overridden text has been changed from `5103 Evidence Notice` to `List of evidence we may need (5103 notice)`.